src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1676,6 +1677,94 @@
               Grid.Row="2"
               Grid.Column="4"
               Grid.RowSpan="2">
+            <TabControl Name="tabDynamic" ItemsSource="{Binding}">
```

Comment:
@reddyashish Can you add some XAML comments to anything worth noticing here?

Reply:
Yes, I will some comments here.